cpp/contracts/xerc20.token.cpp

```diff
@@ -338,5 +346,65 @@ void xtoken::use_burner_limits(xtoken::bridge_model& bridge, const asset& change
    bridge.burning_timestamp = current_block_time().to_time_point().sec_since_epoch();
    bridge.burning_current_limit = current_limit - change;
 }
+
+void xtoken::setfreezeacc(const name& freezing_account) {
```

Comment:
Here the owner can always change the freezing account even after it was setup. This behavior is not the same on evm. Is this wanted?